trackers/react-native-tracker/src/subject.ts

```diff
@@ -0,0 +1,87 @@
+import { CorePluginConfiguration, PayloadBuilder, TrackerCore } from '@snowplow/tracker-core';
+import { ScreenSize, SubjectConfiguration } from './types';
+
+export function newSubject(core: TrackerCore, configuration?: SubjectConfiguration) {
+  let domainUserId: string | undefined;
+  let networkUserId: string | undefined;
+
+  const addSubjectToPayload = (payload: PayloadBuilder): void => {
+    payload.add('duid', domainUserId);
+    payload.add('nuid', networkUserId);
+  };
+
+  const setScreenResolution = (screenSize: ScreenSize) =>
+    core.setScreenResolution(String(screenSize[0]), String(screenSize[1]));
+
+  const setNetworkUserId = function (userId: string | undefined) {
```

Comment:
More of a style nitpick than anything else, but `const foo = function() {}` feels like (in the words of Ryan) a real Hannah Montana moment. I'm not too fussed as long as it's consistent, the `const foo = () => {}` syntax seems to be used above which I see more.